src/services/similarity.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @async
+ * @param {number} recipient_id - The ID of the recipient.
+ * @param {boolean} [cluster] - Specifies whether to cluster the results. Default value is false.
+ * @returns {Promise<Array>} A promise that resolves to an array of similar goals.
+ */
+export default async function getSimilarGoalsForRecipient(recipient_id, cluster) {
+  const { SIMILARITY_ENDPOINT } = process.env;
+
+  const response = await fetch(
+    SIMILARITY_ENDPOINT,
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        recipient_id,
+        cluster,
+        alpha: 0.9,
+      }),
+    },
+  );
+
+  /**
+    * response without clustering looks like:
+    * result: [
```

Comment:
I'm sure it's evident but with clustering is it something like ```[1, 2, 3, 4]```?

Reply:
yeah, just looked more closely at the similarity code, I see what it'll look like